packages/apollo-codegen-swift/src/codeGeneration.ts

```diff
@@ -1045,9 +1063,15 @@ export class SwiftAPIGenerator extends SwiftGenerator<CompilerContext> {
           this.withinBlock(() => {
             this.printOnNewline("get");
             this.withinBlock(() => {
-              this.printOnNewline(
-                `return graphQLMap["${name}"] as! ${typeName}`
-              );
+              if (isOptional) {
+                this.printOnNewline(
+                  `return graphQLMap["${name}"] as? ${typeName}`
+                );
+              } else {
+                this.printOnNewline(
+                  `return graphQLMap["${name}"] as! ${typeName}`
+                );
+              }
             });
```

Comment:
Can be rewritten:
```ts
this.withinBlock(() => 
   this.printOnNewline( isOptional 
       ? `return graphQLMap["${name}"] as? ${typeName}`
       : `return graphQLMap["${name}"] as! ${typeName}`))
```

or even
```ts
this.withinBlock(() => 
   this.printOnNewline(`return graphQLMap["${name}"] as${isOptional ? '?' : '!'} ${typeName}`))
```

IMO this makes it a bit more immediately clear what the effect of `isOptional` is, but this is a style thing that could realistically go either way.

Reply:
This is more in line with how this is done throughout the rest of the file - I like the first option but I'm slightly reluctant to touch the other stuff at the moment if I don't have to.